cmd/cmd.go

```diff
@@ -92,13 +94,86 @@ var newAddress = &cobra.Command{
 	},
 }
 
-
 var burn = &cobra.Command{
-	Use:"burn",
-	Short:"Burns the specied amount of FCT into PNT",
-	Args: CombineCobraArgs(CustomArgOrderValidationBuilder(true, ArgValidatorFCTAddress, ArgValidatorFCTAmount)),
+	Use:   "burn <fct address> <fct amount>",
+	Short: "Burns the specied amount of FCT into PNT",
+	Long: "Burning FCT will turn it into PNT. The PNT burn address is an EC address, and the transaction has " +
+		"an input with # of FCT, and an output of 0 EC. This means the entire tx input becomes the fee. " +
+		"This command costs FCT, so be careful when using it.",
+	Example: "pegnet burn FA3EPZYqodgyEGXNMbiZKE5TS2x2J9wF8J9MvPZb52iGR78xMgCb 1",
+	// TODO: Verify this functionality.
+	ValidArgs: ValidOwnedFCTAddresses(),
+	Args:      CombineCobraArgs(CustomArgOrderValidationBuilder(true, ArgValidatorFCTAddress, ArgValidatorFCTAmount)),
 	Run: func(cmd *cobra.Command, args []string) {
+		defer factom.DeleteTransaction("burn") // Any cleanup from errors
+		// First see if we own the specified FCT address
+		_, err := factom.FetchFactoidAddress(args[0])
+		if err != nil {
+			fmt.Println(err)
```

Comment:
we have a logging tool, let's not use fmt

Reply:
Do we want to use a logger for a cli error? If you run `pegnet random_command`, the error is printed to stderr. I can move the output to stderr vs stdout, but I'd rather not use logrus

Reply:
I used cobra's print error, which defaults to stderr. I also made the exit code 1 on errors. How do you feel about that?

Reply:
logrus outputs everything to stderr (by default) so why couldn't you use logrus for consistency here too?

Reply:
This is a command line feedback... I really am confused.  I've reviewed, but not merged.  

Paul, can you explain how logrus is a help here?  Doesn't its formatting mess up a command line interface?

Reply:
I see logrus as a logging mechanism, the cli errors are not logs. They are a cli failure that should notify the user the command line failed to execute and exit with a non zero status.

Reply:
Ok ok, I finally got what you meant, my mistake here.